astropy/wcs/wcs.py

```diff
@@ -84,7 +84,7 @@
            'NoWcsKeywordsFoundError', 'InvalidTabularParametersError']
 
 
-if six.PY3 or platform.system() == 'Windows':
+if (not six.PY2) or platform.system() == 'Windows':
```

Comment:
Generally I would avoid any unnecessary brackets if the operator precedence is obvious. And since here no `and` and `or` are mixed I think the order of evaluation is pretty obvious, isn't it?